scripts/treasury/cvx_divest_ladder_order.py

```diff
@@ -25,13 +33,29 @@ def rebalance_pool():
         SAFE.init_curve()
         SAFE.curve.deposit(pool, topup, CVX)
     assert BVECVX.balanceOf(pool) == CVX.balanceOf(pool)
-    SAFE.post_safe_tx()
 
 
-def place_orders():
-    SAFE.init_cow()
+def dogfood():
+    # dogfood any crv_bvecvxcvx lp tokens present in the voter msig
+    bal_crvlp = BVECVXCVX.balanceOf(SAFE)
+    if bal_crvlp > 0:
+        BVECVXCVX.approve(BBVECVXCVX, bal_crvlp)
+        BBVECVXCVX.depositAll()
```

Comment:
as now, this will revert since voter is not approved in that sett, added another point in the issue to WL it

Reply:
ref #1042